fedn/fedn/network/combiner/round.py

```diff
@@ -331,13 +331,11 @@ def execute_training_round(self, config):
             model_id = str(uuid.uuid4())
             self.modelservice.set_model(a, model_id)
             a.close()
+            data['model_id'] = model_id
```

Comment:
report server status if model is None? So one can debug...

Reply:
This is reported on line 325.